.travis.yml

```diff
@@ -1,5 +1,7 @@
 sudo: false
 language: go
+go_import_path: github.com/prometheus/procfs
 go:
     - 1.7.6
```

Comment:
I think it'd be reasonable to do 1.x instead of explicit versions here, but will ping others for feedback too.

Reply:
We use stuff like this in other repos
```
go:
- 1.9.x
- 1.x
```